src/App.css

```diff
@@ -19,3 +19,7 @@ body {
   margin: 0;
   padding: 0;
 }
+
+* {
+  font-family: 'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+}
```

Comment:
This was removed from the Homepage.css that got deleted and put into the app.css with the other global css changes like body and root colours.